practices/development.md

```diff
@@ -2,105 +2,90 @@
+Most issues in our repositories are deliverables, in varying states of readiness.
+Deliverables that are added to the [Activity Board](coordination:activity-board) should have enough context and tasks that a team member can realistically close them on their own. [^invest]
 
-A deliverable is ready to work (and can thus be added to the Project Backlog) when it has the following properties (adapted from the [INVEST methodology](https://agileforall.com/new-to-agile-invest-in-good-user-stories/)).
+[^invest]: A good resource for considering what kinds of information makes a deliverable "ready" is [the INVEST methodology](https://agileforall.com/new-to-agile-invest-in-good-user-stories).
 
+:::{admonition} Deliverables should...
+:class: tip
 - **Have a short title and description:** Deliverables should be glanceable and have enough context that the reader can quickly understand the scope.
 - **Have one or more [user stories](https://www.atlassian.com/agile/project-management/user-stories):** User stories should define who benefits from the deliverable, and why they want it.
-- **Have completion criteria:** We have clear completion criteria for this deliverable to denote it as “done”.
 - **Have tasks to complete it**: Deliverables should have a set of tasks, which are actions needed to complete the deliverable.
+:::
 
-### How refinement happens
+(coordination:activity-board)=
+## The Activity Board
 
-_Deliverable Refinement_ is the process of improving the scoping, context, and structure of our Deliverables issues so that they are ready for us to work on them. When a deliverable is created, it may not have all of the information needed to take the next step. Adding that information is the goal of Deliverable Refinement.
+The [Activity Board](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) contains the collection of [Deliverables](coordination:deliverables) that the team is currently working on.
+All deliverables on the Activity Board should be accomplished by the end of the week.
+This board is filled during our [Deliverables review meeting](meetings:deliverables-review).
 
-All team members are expected to participate in deliverable refinement, though the more experienced and higher-level you are, the more you should be contributing to this process.
-The important thing is that we always have a list of high-quality deliverables ready to work towards.
+:::{admonition} The Activity Board should...
+:class: tip
+- Have enough deliverables to keep the team occupied for the week
+- Not have so many deliverables that a team member gets overwhelmed
+- Under-estimate our team's total capacity, to provide room for unexpected work (e.g., support work)
+- Have a team member assigned to each item on the board
+:::
 
-Here's a small table that explains how to decide whether deliverables issues should be put in a project backlog:
+The Activity Board is broken down into these columns:
```

Comment:
If we move to per-week boards, the board should have:

1. Ready to work
2. In progress
3. Done

`Up Next` should be the backlog, and live elsewhere.

Reply:
That makes sense to me (though, we should refine our methods for tracking priorities within a given project, otherwise these meetings are going to involve a lot of "digging through issues" that may not be very efficient, I suspect)

quick question:

What do we do with deliverables that are important to create, but that require more discussion and brainstorming before somebody can just "do them"? An example is https://github.com/2i2c-org/team-compass/issues/175 - that issue still requires discussion, but it's an important discussion to have and does take effort from team members throughout the week. How would we track something like that? (I think in previous iterations this is what would have gone in a "Needs refinement" column).

One option is that we follow the same workflow, but use a github label like `needs:decision`, and the person "assigned" to the issue is in charge of moving forward the discussion and converging on a decision.

Reply:
> What do we do with deliverables that are important to create, but that require more discussion and brainstorming before somebody can just "do them"?

Reply:
yeah I think the challenge is that it is harder to scope cards that need more discussion etc, since it requires more input from others and such. That said, I agree with your proposal here so let's just give this a shot and see if it's awkward in some spaces that make us want to update